src/lib/pubkey/xmss/xmss.h

```diff
@@ -250,9 +274,13 @@ class BOTAN_PUBLIC_API(2,0) XMSS_PrivateKey final : public virtual XMSS_PublicKe
        **/
       secure_vector<uint8_t> raw_private_key() const;
 
+      WOTS_Derivation_Method wots_derivation_method() const { return m_wots_derivation_method; }
+
    private:
       friend class XMSS_Signature_Operation;
 
+      XMSS_PrivateKey(std::span<const uint8_t> raw_key, bool is_legacy_key);
+
```

Comment:
That's a left-over from the previous iteration (using the `is_legacy_key` boolean). It should go, because it doesn't have an implementation and is unused.

Reply:
Ops, I missed that...